tools/build_script_generator/cmake/resources/Makefile.in

```diff
@@ -59,6 +59,7 @@ program-bmp: build
 ui?=tui
 debug: build
 	@python3 modm/modm_tools/gdb.py -x modm/gdbinit -x modm/openocd_gdbinit \
+			-ex "dir $(dir $(realpath $(dir $(realpath $(shell which arm-none-eabi-gcc)))))" \
```

Comment:
FYI: This Makefile will get a make over via the `::make` generator, so this is currently more of an ugly placeholder.